h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -1,6 +1,9 @@
 import { useId, useState } from 'preact/hooks';
 
 import { Button, Input, Textarea } from '@hypothesis/frontend-shared';
+import readConfig from '../config';
```

Comment:
I missed this before, but `readConfig` should be a non-default export. The convention in our code is to use default exports only for Preact components. This tends to avoid accidentally "renaming" symbols (eg. defining the function as `getConfig` inside `config.ts` and importing it as `readConfig`).

Reply:
Done